doc/setup.md

```diff
@@ -79,15 +79,32 @@ sudo service postgresql restart
 - Extract the binary to somewhere on your path
 - Ensure it runs with `geckodriver --version`
 
+4. make the dependencies using
+
+```
+make dependencies
+```
+
 ## Launch
 
 1. Initialize the database with `make db_init` (this only needs to
    happen once).
 2. Copy `config.yaml.defaults` to `config.yaml`.
-2. If you want some test data to play with, run `make load_demo_data`.
 3. Run `make log` to monitor the service and, in a separate window, `make run` to start the server.
+4. Direct your browser to http://localhost:5000. Login to SkyPortal.
+5. If you want some test data to play with, run `make load_demo_data` (do this while the server is running!).
+6. Change users by navigating to localhost:5000/become_user/<#> where # is a number from 1-5.
+   Different users have different privileges and can see more or less of the demo data.
+
+## Troubleshooting
+
+If you have trouble getting the demo data try doing
+
+```make db_clear && make db_init && make run```
+
+and then, after logging in, from a different window, do `make load_demo_data`.
 
-Direct your browser to http://localhost:5000.
+You should be able to log in as testadmin rather than testuser.
```

Comment:
Not sure what this instruction is meant to tell the reader.

Reply:
Ok I'm not sure either. Will remove it...